web/app/js/components/HealthPane.jsx

```diff
@@ -26,7 +26,10 @@ export default class HealthPane extends React.Component {
     return _.meanBy(metrics, 'successRate');
   }
 
-  getHealthClassName(successRate) {
+  getHealthClassName(successRate, deploymentAdded) {
+    if (!deploymentAdded) {
```

Comment:
let's just use `this.props.deploymentAdded` here rather than passing it in as an argument, since `getHealthClassName` is in the component.
(there's also an argument for moving this function somewhere generic like MetricUtils, in which case we could keep passing it in).

yet another thing we could do is use `null` or `undefined` to indicate lack of a value (as opposed to 0 indicating a zero value) and use that in the determination. maybe it's most clear if we do use deploymentAdded as an indicator though.